src/components/mock_location.tsx

```diff
@@ -0,0 +1,237 @@
+import { locations } from '../interfaces/location.interfaces'
```

Comment:
I think your locations interface type should has the same convention with other Interface